### PR TITLE
Migration to mainnet

### PR DESCRIPTION
**Pull request by josojo**
This is only the migration to mainnet.

I just ran:
```
//Mainnet deployment:  
export NETWORK_NAME=mainnet 
export GAS_PRICE_GWEI=13 
export PK=$PRIVATE_KEY_GNOSIS
  rm -r build 
npx truffle compile 
npm run networks-inject
npx truffle migrate --network $NETWORK_NAME --onlyMigrateStableX=true 
npm run networks-extract
npx truffle run verify BatchExchange --network $NETWORK_NAME  
//If it does not work, do it manually using the flattener: npx truffle-flattener contracts/BatchExchange.sol > flattend.txt  
// commit new network.json file
npm version patch -m “New deployment"
npm publish
 //Adding tokens to exchange
npx truffle exec scripts/stablex/add_token_list.js --network $NETWORK_NAME

// Place orders:
 // Depositing OWL:
 npx truffle exec scripts/stablex/deposit.js --accountId=0 --tokenId=0 --amount=30 --network $NETWORK_NAME 
// Getting TrueUSD
npx truffle exec scripts/stablex/invokeViewFunction.js 'tokenAddressToIdMap' '0x0000000000085d4780B73119b644AE5ecd22b376' --network $NETWORK_NAME 
export TOKEN_ID_TUSD=3 
// Depositing TrueUSD
npx truffle exec scripts/stablex/deposit.js --accountId=0 --tokenId=$TOKEN_ID_TUSD --amount=98 --network $NETWORK_NAME  
// Getting DAI npx truffle exec scripts/stablex/invokeViewFunction.js 'tokenAddressToIdMap' '0x6b175474e89094c44da98b954eedeac495271d0f' --network $NETWORK_NAME 
export TOKEN_ID_DAI=8
// Depositing DAI
npx truffle exec scripts/stablex/deposit.js --accountId=0 --tokenId=$TOKEN_ID_DAI --amount=100 --network $NETWORK_NAME  
// OWL vs DAI order: npx truffle exec scripts/stablex/place_order.js --accountId=0 --buyToken=$TOKEN_ID_DAI --sellToken=0 --minBuy=1000 --maxSell=1000 --validFor=2000 --network $NETWORK_NAME

 // Market maker order TUSD vs DAI
npx truffle exec scripts/stablex/place_order.js --accountId=0 --buyToken=$TOKEN_ID_DAI --sellToken=$TOKEN_ID_TUSD --minBuy=1000 --maxSell=998 --validFor=200 --network  $NETWORK_NAME

// Market maker order DAI vs TUSD
npx truffle exec scripts/stablex/place_order.js --accountId=0 --buyToken=$TOKEN_ID_TUSD --sellToken=$TOKEN_ID_DAI --minBuy=1000 --maxSell=998 --validFor=200 --network  $NETWORK_NAME
```

nit: due to the fact that SAI was in the tokenList and not DAI for mainnet, I needed to do some extra work. Now both SAI and DAI are listed. I submitted PR to dex-react to correct this error.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -59,7 +59,7 @@
   "devDependencies": {
     "@gnosis.pm/mock-contract": "^3.0.8",
     "@gnosis.pm/owl-token": "^3.1.0",
-    "@gnosis.pm/solidity-data-structures": "^1.1.0",
+    "@gnosis.pm/solidity-data-structures": "^1.2.1",
```
Comment (fleupold) [inline package.json:62]: In the other PR (https://github.com/gnosis/solidity-data-structures/pull/8/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R3) it says 1.2.0

**Review by fleupold — APPROVED** [review]
Awesome! Do we need to push another npm package of this one?

**Comment by fleupold** [comment]
Also noting the commands you ran in the PR description might be helpful for anyone that might have to do the migration in the future.

**Review by anxolin — APPROVED** [review]
Nicccccccce!, let us know the npm package once it's published 👍 

We are in mainnet <a href="https://gitme.me/image?url=https%3A%2F%2Fmedia2.giphy.com%2Fmedia%2Fl0MYt5jPR6QX5pnqM%2Fgiphy.gif&token=party" data-gitmeme-token="party"></a>y

File: package.json (hunk 1 above)
Comment (josojo) [reply package.json:62]: no longer

**Comment by anxolin** [comment]
By the way, mac users, something interesting for the passwords we can use:

```bash
export PLAYGROUND=$(security find-generic-password -a ${USER} -s playground -w)
```

extracted from https://www.netmeister.org/blog/keychain-passwords.html

It uses the apple keychain app. I wouldn't add a super important mnemonic, but interesting for many of the mnemonics we use.

**Pull request merged**